website/src/content/api-reference/commercetools-frontend-actions-global.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: "@/actions-global"
+---
+
+<Subtitle>
+
+Global Redux actions and hooks for handling user notifications.
+
+</Subtitle>
+
+# Installation
+
+```
+yarn add @commercetools-frontend/actions-global
+
+# or
+
+npm --save install @commercetools-frontend/actions-global
+```
+
+Additionally install the peer dependencies (if not present)
+
+```
+yarn add react redux react-redux
+
+# or
+
+npm --save install react redux react-redux
+```
+
+# Notification system
+
+Custom Applications come with a built-in notification system built on top of [Redux](https://redux.js.org).
+
+The `@commercetools-frontend/actions-global` package provides Redux action creators and React hooks to facilitate dispatching notifications within Custom Applications.
+
+# Action creators
+
+<Info>
+
+Rather than using the action creators directly we recommend using the corresponding React hooks. The hooks have the `dispatch` method of the Redux store bound internally. 
+
+</Info>
+
+## showNotification
+
+An action creator to dispatch any kind of notification.<br />
+Takes 2 arguments: `notification` object (required) and `meta` object (optional).
```

Comment:
I'm confused about `text` not being required since I can't see a use case when we want to show a notification with no text.

Reply:
+1

Reply:
Ah! understood.
I was expecting the user to always provide the message, but if we already provide that behaviour, me might want to keep it.

Thanks for the explanation 👍